src/pages/Dashboard/Markets/BorrowMarket/BorrowMarketTable.tsx

```diff
@@ -64,6 +75,21 @@ const BorrowMarketTable: React.FC<IBorrowMarketTableProps> = ({
     ];
   });
 
+  if (isSmDown) {
+    return (
+      <>
+        <h4>{t('markets.borrowMarketTableTitle')}</h4>
+        {rows.map(([asset, apy, wallet, liquidity]) => (
+          <AssetCardMobile
+            title={asset.render()}
+            columns={columnsMobile}
+            data={[[apy, wallet, liquidity]]}
+            rowOnClick={rowOnClick}
+          />
+        ))}
+      </>
```

Comment:
I'd approach the problem differently. Since all our tables use the card layout on mobile, I think we should update the `Table` component so it automatically switches to the correct layout based on the resolution.

Also, please use CSS as much as possible to handle responsiveness rather than the JS hooks.